lib/src/mock.dart

```diff
@@ -1270,3 +1270,27 @@ extension ListOfVerificationResult on List<VerificationResult> {
   /// [verifyInOrder].
   List<List<dynamic>> get captured => [...map((result) => result.captured)];
 }
+
+final _defaultDummyValueFor = <Type, dynamic>{
+  num: 0,
+  int: 0,
+  double: 0.0,
+  bool: false,
+  String: '',
+};
+
+void setDefaultDummyValue<T>(T value) {
```

Comment:
To prevent accidentally calling `defaultDummyValue()` (without an explicit type argument), I think we should throw if `T` is `dynamic`. (Or, if a bound is added, maybe throw if `T` is `Object`?)